obspy/signal/tests/test_trigger.py

```diff
@@ -9,14 +9,33 @@
 from ctypes import ArgumentError
 
 import numpy as np
+from numpy.testing import assert_array_almost_equal, assert_array_equal
```

Comment:
@d-chambers we got rid of the numpy.testing methods, right ?

Reply:
Why? They are so handy.

Reply:
You can do either one, both result in a test failure with an informative message. We just want to avoid anything unittest related.